legal-api/report-templates/template-parts/common/addresses.html

```diff
@@ -0,0 +1,87 @@
+<div class="address-container">
+   {% if offices.registeredOffice is defined %}
+   <div class="section-title">
+      Registered Office Information
+   </div>
+   <div class="section-data">
+      <table class="section-data-table" role="presentation">
+         <tr>
+            <td class="delivery-address">
```

Comment:
Per comments in /bcgov/entity#4230, mailing address should come first, then delivery address.